### ISSUE TITLE
artisan orion:specs does not work

### ISSUE THREAD

**Issue by tndaniel05** [tailflow/laravel-orion#244]
```
php artisan orion:specs

   ERROR  There are no commands defined in the "orion" namespace. 
```

This is what I got when I try to generate the specs. Can you help me what's wrong? 
Thank you.

**Comment by ldiebold** [comment]
Check your composer.json file. Are you using Orion V2?
I've noticed that sometimes, even on a new project, orion installs V1 by default.

**Comment by tndaniel05** [comment]
I thought it's the latest version because composer upgrade did not do anything.
But you are right, in the composer.json it was on V1.

The issue was the following:
Problem 1
    - Root composer.json requires tailflow/laravel-orion ^2.22.1 -> satisfiable by tailflow/laravel-orion[2.22.1].
    - tailflow/laravel-orion 2.22.1 requires symfony/yaml ^5.3|^6.0 -> found symfony/yaml[v5.3.0, ..., v5.4.35, v6.0.0, ..., v6.4.3] but the package is fixed to v7.0.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

So I had to downgrade symfony/yaml to 6.4.3 to install V2 Orion. 

Thank you for the help!
